COMFIE/Presentation/Memo/MemoView.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MemoView.swift
+//  COMFIE
+//
+//  Created by Anjin on 3/6/25.
+//
+
+import SwiftUI
+
+struct MemoView: View {
+    @State var intent: MemoStore
```

Comment:
```suggestion
    @State var store: MemoStore
```
변수명을 더 직관적으로 만들어보는 건 어떨까요?

Reply:
흠.. 이건 저도 고민을 많이 했던 부분인데요..
```swift
struct OnboardingView: View {
    @State var intent: OnboardingStore  // View에서 일어나는 User의 의도
    private var state: OnboardingStore.State { intent.state }  // View에서 보여지는 상태값
    
    var body: some View {
        VStack {
            Text("온보딩 View")
            
            Button {
                intent(.nextButtonTapped)
            } label: {
                Text("메모 화면으로")
            }
        }
//        .environment(intent)
    }
}
```
View에서 쓰여질때, store에 intent를 실행시킨다는 의도로 표현되는 것이 좋을지,
아니면 View에서 intent를 트리거한다는 의도로 표현되는 것이 좋을지 고민했습니다!

위에 언급한 코드에서 처럼 View에서 보여지는 state와 의도 intent를 표현하는 것은 어떨지 생각했어요! 현재는 보여지는 State값이 없어서 따로 state를 선언하진 않았지만 위 코드처럼 쓰이면 어떨까 하던 생각에서 intent로 쓰게 되었습니다.

그치만 이야기한것처럼 언제든 우리가 더 소통하기 좋고 편한 것으로 쓰면 좋을 것 같아요!!
@zaehorang 의견 부탁!

Reply:
아항 이오의 얘기를 들어보니 `View에서 Intent를 트리거한다`는 의도를 명시해주는 게 좋아보이네요! 
현재 네이밍으로 진행해도 좋아 보입니다 !